plugins/node/opentelemetry-instrumentation-aws-lambda/src/aws-lambda.ts

```diff
@@ -24,21 +24,47 @@ import {
   safeExecuteInTheMiddle,
 } from '@opentelemetry/instrumentation';
 import {
+  Context as OtelContext,
+  context as otelContext,
```

Comment:
Why the two different `context`s here?

Reply:
There's `context`, an object with methods exposing the context API of type `ContextAPI`. Then there's `Context`, which is the type of an actual passed around context. It would be nice I think if `context` could be renamed `contextAPI` or similar.